aisdc/metrics.py

```diff
@@ -136,17 +213,12 @@ def auc_p_val(auc: float, n_pos: int, n_neg: int) -> tuple[float, float]:
     return auc_p, auc_std
 
 
-def get_metrics(  # pylint: disable=too-many-locals
-    clf,
-    X_test: np.ndarray,
-    y_test: np.ndarray,
-    permute_rows: bool = True,
+def get_probabilities(  # pylint: disable=too-many-locals
+    clf, X_test: np.ndarray, y_test: np.ndarray = None, permute_rows: bool = None
```

Comment:
Shouldn't `permute_rows` be default set to `False` and not `None` ? None is not a bool?

Reply:
also `y_test: np.ndarray = None` -- None is not an ndarray